what-is-chalk.mdx

```diff
@@ -18,215 +18,481 @@ import { PyDiffEditor } from '@/components/Editor'
+- Export features to catalogs like Iceberg for downstream analytics and usage
+- Trace data lineage all the way down to source tables
 
-<RequestingFeaturesOnline style={{ height: 328 }} />
+```python
+from chalk.client import ChalkClient
+
+client = ChalkClient(branch=True)
+client.load_features() # load prod features with one line
 
-Features computed to serve online requests are also replicated to Chalk's
-offline store for historical performance tracking and training set generation.
+User.name_exclaimed = _.name + "!" # add new features
 
-### Offline training
+chalk_dataset = client.offline_query(
+    input={
+        User.id: list(range(1000)),
+    },
+    output={
+        User.id,
+        User.name,
+        User.name_exclaimed,
+    },
+    recompute_features=True, # A/B test against historical model runs
+    dataset_name="fraud_model",
+)
 
-Data scientists can use Chalk's Jupyter integration to create datasets
-and train models. Datasets are stored and tracked so that they can be
-re-used by other modelers. Chalk implements model provenance to track
-inputs, outputs, and other data for audit and reproducibility.
+df = chalk_dataset.to_pandas() # convert to pandas dataframe
 
-<JupyterNotebook />
+# write to Glue or Iceberg
+catalog = GlueCatalog(
+    name="aws_glue_catalog",
+    aws_region="us-west-2",
+    catalog_id="123",
+    aws_role_arn="arn:aws:iam::123456789012:role/OurCatalogueAccessRole",
+)
+chalk_dataset.write_to(destination="database.table_name", catalog=catalog)
+```
 
-Chalk datasets are always [temporally consistent](/docs/temporal-consistency).
-This means that you can provide labels with different past timestamps to easily
-get historical features that represent what your application would have
-retrieved online at those past times. Temporal consistency ensures that
-your model training doesn't mix "future" and "past" data.
+Chalk evaluates features with point-in-time lookups, guaranteeing evaluation only with data that would have been seen in the past. This means that you can provide labels with different past timestamps to easily get historical features that represent what your application would have retrieved online at those past times.
 
 <TemporalCorrectness />
+
+With Chalk, data scientists easily integrate new data sources, test new features with [point-in-time correctness,](docs/temporal-consistency) and collaborate with Git-like branches and flexible data exports (to data catalogs, Parquet, Iceberg, etc.).
+
+---
+
+## Chalk for Data Engineers
+
+Manage features declaratively without the complexity of orchestrating ETL jobs, feature stores, and offline/online datastores.
+Chalk makes it easy to
+
+- Cache expensive computations with configurable staleness
+- Manage model versions and A/B test features
+- Create complex joins and relationships between data entities
+- Configure time-window aggregations with flexible materialization options
```

Comment:
i think we probably have good links for all of these!

Reply:
added links